### PR TITLE
Mcafee tie playbook update

### PR DESCRIPTION
**Pull request by guydemi**
updated to use the wizard instead of dt when possible,
added outputs,
added test playbook

relates to: https://github.com/demisto/etc/issues/8532
relates to: https://github.com/demisto/etc/issues/10411

### CODE REVIEW

File: Playbooks/playbook-IOC_Hunt_-_TIE.yml [hunk 1]
```diff
@@ -0,0 +1,303 @@
+id: IOC Hunt - TIE
+version: -1
+name: IOC Hunt - TIE
```
Comment (anara123) [inline Playbooks/playbook-IOC_Hunt_-_TIE.yml]: leave the old id. For backward compatability

**Comment by anara123** [comment]
Your test fails because you missing descriptions, please make sure the build is green

**Comment by anara123** [comment]
@guydemi also attach screenshot of the IOC Hunt - TIE playbook and the screenshot of test playbook.

**Comment by anara123** [comment]
image
Why you add face hostname?

**Comment by asafshen** [comment]
@guydemi 
we have made some changes in playbook structure - please download the playbook again from an updated demisto env (demo2/your local env)

File: Playbooks/playbook-IOC_Hunt_-_TIE.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (guydemi) [reply Playbooks/playbook-IOC_Hunt_-_TIE.yml]: so what will happen when less than 3.5.0?
maybe duplicate and use 2 different playbooks? (one unchanged - for backward compatibility, and one new with "fromVersion")

**Comment by guydemi** [comment]
@anara123, I have added the fake hostname because we don't have a working ePo instance, so hostname will never return while testing

**Comment by guydemi** [comment]
ioc_hunt_-_tie_sun_jan_21_2018
ioc_hunt_-_tie_-_test_sun_jan_21_2018

File: Tests/conf.json [hunk 2]
```diff
@@ -165,6 +165,9 @@
         },
         {
             "playbookID": "endpoint_enrichment_-_generic_test"
+        },
+        {
+            "playbookID": "IOC Hunt - TIE - Test"
```
Comment (anara123) [inline Tests/conf.json]: You need to add which integration playbook will use `McAfee Threat Intelligence Exchange`

**Pull request merged**